src/overlays/actors/ovl_En_Bjt/z_en_bjt.c

```diff
@@ -15,10 +15,37 @@ void EnBjt_Destroy(Actor* thisx, PlayState* play);
 void EnBjt_Update(Actor* thisx, PlayState* play);
 void EnBjt_Draw(Actor* thisx, PlayState* play);
 
-void func_80BFDA48(EnBjt* this, PlayState* play);
-void func_80BFDAE8(EnBjt* this, PlayState* play);
+void EnBjt_Talk(EnBjt* this, PlayState* play);
+void EnBjt_Wait(EnBjt* this, PlayState* play);
+
+#define TOILET_HAND_STATE_TALKING (1 << 3)   // Actually talking to Player
+#define TOILET_HAND_STATE_TEXTBOX (1 << 4)   // whenever a textbox is on screen
+#define TOILET_HAND_STATE_APPEARING (1 << 7) // Appearing
+#define TOILET_HAND_STATE_VANISHING (1 << 8) // Vanishing
+#define TOILET_HAND_STATE_VISIBLE (1 << 9)   // Out
```

Comment:
Why there's no defines for `(1<<0)`, `(1<<1)` and `(1<<2)`? I see those are getting checked in many places, those are also set and unset with `SubS_UpdateFlags`

Reply:
Yeah, since they only seem to be used with SubS_UpdateFlags they seem to only be part of that system, so as I said in the original PR message I think it's out of scope for this PR to try to work those out.